<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CitationBibliographyWidget</class>
 <widget class="QDialog" name="CitationBibliographyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>561</height>
    </rect>
   </property>
   <property name="title">
    <string>Insert Bibliography Entry Details</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>31</y>
      <width>711</width>
      <height>528</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="shortnameLabel">
       <property name="text">
        <string>Short Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="shortName">
       <property name="text">
        <string>DefaultTag1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="author"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="title"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Year:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="year"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="publisherLabel">
       <property name="text">
        <string>Publisher:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="publisher"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="address"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="isbnLabel">
       <property name="text">
        <string>ISBN:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="isbn"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="chapterLabel">
       <property name="text">
        <string>Chapter:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="chapter"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="pagesLabel">
       <property name="text">
        <string>Page(s):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="pages"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="editorLabel">
       <property name="text">
        <string>Editor:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="editor"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="editionLabel">
       <property name="text">
        <string>Edition:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="edition"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="booktitleLabel">
       <property name="text">
        <string>Book title:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="booktitle"/>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="volumeLabel">
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="volume"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="publicationLabel">
       <property name="text">
        <string>Publication:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="publication"/>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="organisationLabel">
       <property name="text">
        <string>Organisation:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLineEdit" name="organisation"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="institutionLabel">
       <property name="text">
        <string>Institution:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="institution"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="schoolLabel">
       <property name="text">
        <string>University:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLineEdit" name="school"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="reporttypeLabel">
       <property name="text">
        <string>Type of Report:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="reporttype"/>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="monthLabel">
       <property name="text">
        <string>Month:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLineEdit" name="month"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="journalLabel">
       <property name="text">
        <string>Journal:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="journal"/>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="numberLabel">
       <property name="text">
        <string>Number:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QLineEdit" name="number"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="seriesLabel">
       <property name="text">
        <string>Series:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="series"/>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="annotationLabel">
       <property name="text">
        <string>Annotation:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QLineEdit" name="annotation"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="noteLabel">
       <property name="text">
        <string>Note:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="note"/>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QLineEdit" name="url"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="ud1Label">
       <property name="text">
        <string>User-defined 1:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="ud1"/>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="ud2Label">
       <property name="text">
        <string>User-defined 2:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QLineEdit" name="ud2"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="ud3Label">
       <property name="text">
        <string>User-defined 3:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="ud3"/>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="ud4Label">
       <property name="text">
        <string>User-defined 4:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QLineEdit" name="ud4"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="ud5Label">
       <property name="text">
        <string>User-defined 5:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="ud5"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="sourcetypeLabel">
       <property name="text">
        <string>Source:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="sourceType">
       <item>
        <property name="text">
         <string>Article</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Book</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Conference proceedings</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Book excerpts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Book excerpts with title</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Journal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Techn. documentation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thesis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Miscellaneous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dissertation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Research report</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unpublished</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Email</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WWW document</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User defined 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User defined 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User defined 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User defined 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User defined 5</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>630</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>701</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>From the document:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="existingCites"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>shortName</tabstop>
  <tabstop>sourceType</tabstop>
  <tabstop>author</tabstop>
  <tabstop>title</tabstop>
  <tabstop>year</tabstop>
  <tabstop>publisher</tabstop>
  <tabstop>address</tabstop>
  <tabstop>isbn</tabstop>
  <tabstop>chapter</tabstop>
  <tabstop>pages</tabstop>
  <tabstop>editor</tabstop>
  <tabstop>edition</tabstop>
  <tabstop>booktitle</tabstop>
  <tabstop>volume</tabstop>
  <tabstop>publication</tabstop>
  <tabstop>organisation</tabstop>
  <tabstop>institution</tabstop>
  <tabstop>school</tabstop>
  <tabstop>reporttype</tabstop>
  <tabstop>month</tabstop>
  <tabstop>journal</tabstop>
  <tabstop>number</tabstop>
  <tabstop>series</tabstop>
  <tabstop>annotation</tabstop>
  <tabstop>note</tabstop>
  <tabstop>url</tabstop>
  <tabstop>ud1</tabstop>
  <tabstop>ud2</tabstop>
  <tabstop>ud3</tabstop>
  <tabstop>ud4</tabstop>
  <tabstop>ud5</tabstop>
  <tabstop>existingCites</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CitationBibliographyWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
